docs/connection-api.md

```diff
@@ -1,7 +1,7 @@
-# Connection API
+# Connection APIs
 
 * [Main API](#main-api)
-* [`Connection` API](#connection-api-1)
+* [`Connection` API](#connection-api)
```

Comment:
please make sure to check links in the site once its merged

Reply:
I will check the links tomorrow.. however the IDE did not show broken links.. but better safe than sorry I guess